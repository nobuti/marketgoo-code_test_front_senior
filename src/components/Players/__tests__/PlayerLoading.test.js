import React from 'react'
import { render, screen } from '@testing-library/react'
import Loading from '../PlayersLoading'

describe('PlayersLoading', () => {
  test('renders properly', () => {
    const { container } = render(
      <Loading />
    )

    expect(container).toMatchSnapshot()
    expect(screen.queryByTestId('loading')).toHaveClass('Loading')
    expect(screen.queryByTestId('loading-label')).toHaveClass('Loading-label')
  })
})
