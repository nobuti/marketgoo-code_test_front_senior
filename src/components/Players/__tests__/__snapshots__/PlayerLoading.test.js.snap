// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayersLoading renders properly 1`] = `
<div>
  <div
    class="Loading"
    data-testid="loading"
  >
    <svg
      class="ola_spinner"
      height="28"
      viewBox="0 0 28 28"
      width="28"
    >
      <circle
        cx="14"
        cy="14"
        r="12"
      />
    </svg>
    <span
      class="Loading-label"
      data-testid="loading-label"
    >
      Loading players
    </span>
  </div>
</div>
`;
